src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -126,6 +129,23 @@ static DatabaseHandler()
 
             displaySizeBasedCleanupMode.TryAdd(QueryStoreSizeBasedCleanupMode.Off, SR.queryStoreSizeBasedCleanupMode_Off);
             displaySizeBasedCleanupMode.TryAdd(QueryStoreSizeBasedCleanupMode.Auto, SR.queryStoreSizeBasedCleanupMode_Auto);
+            displayRecoveryStateOptions = new CategoryValue[]{
+                new CategoryValue
+                {
+                    Name = "WithRecovery",
+                    DisplayName = "RESTORE WITH RECOVERY"
```

Comment:
Consider adding comments explaining that these are T-SQL keywords so shouldn't be localized